Description: Drop "image2" test
 This test uses an image which has been excluded from the
 package sources as it isn't licensed under a DFSG-compatible
 license.
Author: Arnaud Ferraris <aferraris@debian.org>
Last-Update: 2023-11-03

--- a/tests/test.rs
+++ b/tests/test.rs
@@ -29,20 +29,3 @@
     assert_eq!(colors[7], Color::new(167, 199, 221)); // 167, 198, 220
     assert_eq!(colors[8], Color::new(212,  80,   7)); // 213,  75,   8
 }
-
-#[test]
-fn image2() {
-    let img = image::open(&path::Path::new("images/iguana.png")).unwrap();
-    let color_type = find_color(img.color());
-    let colors = color_thief::get_palette(&img.raw_pixels(), color_type, 10, 10).unwrap();
-
-    assert_eq!(colors[0], Color::new( 71,  60,  53));
-    assert_eq!(colors[1], Color::new(205, 205, 202));
-    assert_eq!(colors[2], Color::new(165, 170, 174));
-    assert_eq!(colors[3], Color::new(147, 137, 129));
-    assert_eq!(colors[4], Color::new(146, 152, 168));
-    assert_eq!(colors[5], Color::new(117, 122, 128));
-    assert_eq!(colors[6], Color::new(100, 101, 113));
-    assert_eq!(colors[7], Color::new( 22,  20,  27));
-    assert_eq!(colors[8], Color::new(180, 148, 116));
-}
